apiserver/restrict_controller.go

```diff
@@ -27,6 +27,7 @@ var controllerFacadeNames = set.NewStrings(
 var commonFacadeNames = set.NewStrings(
 	"Pinger",
 	"Bundle",
+	"HighAvailability", // Exposed for model logins for backwards compatibility.
```

Comment:
Can you make this a bug TODO to fix once we very that no external clients use it. Target the bug to 2.0.1